### ISSUE TITLE
class:move to another namespace, missing use statements

### ISSUE THREAD

**Issue by umpirsky** [phpactor/phpactor#265]
Here is the case, let's say we have class `Foo/Bar`, and we want to move it to `Foo2/Bar2`, but we also have `Foo/Baz` class that have:

```php
/** @var Bar */
    private $bar;
```

It didn't need use statement before the move, but it needs after. Not sure if it does not add use statement just because it's annotation?

**Comment by dantleech** [comment]
Yeah, it's not aware of annotations at the moment, maybe you could change the title of the ticket to the form of a feature request?
